README.md

```diff
@@ -265,18 +314,26 @@ $ gto which churn prod --ref
 churn#prod#2
 ```
 
+<details summary="Kanban/Stages workflow">
+If you prefer Kanban/Stages workflow described above, you could use `--last` flag:
+
+```console
+$ gto which churn prod --last
+v3.1.0
```

Comment:
From the example workflow above, I don't see why `gto which churn prod --last` would return `v3.1.0` since `v3.1.0` was never assigned label `prod`.

What happens in this scenario:
1. I assign v3.0.0 to staging.
2. I assign v3.1.0 to staging.
3. I assign v3.1.0 to prod.

What should `gto which churn staging` return? What should `gto which churn staging --last` return? What about `gto show` with and without `--last`?

Reply:
> What happens in this scenario

I assume that those are only assignments that were executed to `churn`. then:
```shell
$ gto which churn staging
v3.1.0
$ gto which churn staging --last
v3.0.0
$ gto show
 ╒═══════════════╤══════════╤════════╤═════════╤════════════╕
 │ name          │ latest   │ #dev   │ #prod   │ #staging   │
 ╞═══════════════╪══════════╪════════╪═════════╪════════════╡
 │ churn         │ v3.1.0   │ -      │ v3.1.0  │ v3.1.0     │
 ╘═══════════════╧══════════╧════════╧═════════╧════════════╛
$ gto show --last
 ╒═══════════════╤══════════╤════════╤═════════╤════════════╕
 │ name          │ latest   │ #dev   │ #prod   │ #staging   │
 ╞═══════════════╪══════════╪════════╪═════════╪════════════╡
 │ churn         │ v3.1.0   │ -      │ v3.1.0  │ v3.0.0     │
 ╘═══════════════╧══════════╧════════╧═════════╧════════════╛
```